services/urls.py

```diff
@@ -18,6 +18,12 @@
         view=views.ServiceDetail.as_view(),
         name='detail'
     ),
+    # URL pattern for the ServiceDetailView --> Get Services by Category
+    url(
+        regex=r'^feed$',
+        view=views.CategoryServiceList.as_view(),
+        name='detail'
```

Comment:
This name should be 'feed'. 

This names are used to reference this entries. Is like the variable name of this URL entry.